### DISCUSSION TITLE
Serverless deploy error with docker image

### DISCUSSION THREAD

**Discussion by pawanreddy1708** [serverless/serverless#11792] [Q&A]
### Are you certain it's a bug?

- [X] Yes, it looks like a bug

### Is the issue caused by a plugin?

- [X] It is not a plugin issue

### Are you using the latest v3 release?

- [X] Yes, I'm using the latest v3 release

### Is there an existing issue for this?

- [X] I have searched existing issues, it hasn't been reported yet

### Issue description

Hi Team,

I ran serverless deploy for a tiny project, i ran into docker issue, that says it needs specific python-build3.9, i tried pulling the available build on docker hub this still did not resolve the issue.

Running docker run --rm -v /Users/pawreddy/Library/Caches/serverless-python-requirements/4de54344cbea6e69d798737ff8ab8b9b3bd2d640186abfcc7a8bb07a6b8b9564_x86_64_slspyc\:/var/task\:z -v /Users/pawreddy/Library/Caches/serverless-python-requirements/downloadCacheslspyc\:/var/useDownloadCache\:z -u 0 lambci/lambda\:build-python3.9 python3.9 -m pip install -t /var/task/ -r /var/task/requirements.txt --cache-dir /var/useDownloadCache...
Stdout:
Stderr: Unable to find image 'lambci/lambda:build-python3.9' locally
docker: Error response from daemon: manifest for lambci/lambda:build-python3.9 not found: manifest unknown: manifest unknown.
See 'docker run --help'.

### Service configuration (serverless.yml) content

```yaml
N?A
```

### Command name and used flags

serverless deploy --stage dev --verbose

### Command output

```shell
Running docker run --rm -v /Users/pawreddy/Library/Caches/serverless-python-requirements/4de54344cbea6e69d798737ff8ab8b9b3bd2d640186abfcc7a8bb07a6b8b9564_x86_64_slspyc\:/var/task\:z -v /Users/pawreddy/Library/Caches/serverless-python-requirements/downloadCacheslspyc\:/var/useDownloadCache\:z -u 0 lambci/lambda\:build-python3.9 python3.9 -m pip install -t /var/task/ -r /var/task/requirements.txt --cache-dir /var/useDownloadCache...
Stdout:
Stderr: Unable to find image 'lambci/lambda:build-python3.9' locally
docker: Error response from daemon: manifest for lambci/lambda:build-python3.9 not found: manifest unknown: manifest unknown.
See 'docker run --help'.
```

### Environment information

```shell
Framework Core: 3.28.1 (local)
Plugin: 6.2.3
SDK: 4.3.2
```

**Comment by medikoo** [comment]
@pawanreddy1708 that's more a support request than a bug report. Best is to rely on stack overflow for such inquiries I'm moving it to discussions

**Reply by pawanreddy1708** [reply]
Hey I tried there, i could not find any related ?
Could you give me some leads on this.

**Comment by joshdick** [comment]
@medikoo I'm running into the same problem. It looks like this is a real issue because `lambci/docker-lamba` currently referenced by `serverless` has been deprecated: https://github.com/lambci/docker-lambda

Edit: I was able to work around this issue by using the workaround mentioned in https://github.com/lambci/lambci/issues/138#issuecomment-1193211670 (an issue accidentally filed in an unrelated project with a similar name) in combination with the official AWS image (comment directly underneath the linked one.)

**Reply by medikoo** [reply]
@joshdick thanks for input. If you have a good idea on how `lambci/docker-lamba` can be replaced with something maintained in the Framework can you open an issue in the Framework with proposal?
